modules/oe_content_call_proposals/README.md

```diff
@@ -0,0 +1,51 @@
+# OpenEuropa Call for proposals Content
+
+This module provides the Call for proposals content type.
+
+## Installation
+
+Before enabling this module, make sure the following modules are present in your codebase by adding them to your
+`composer.json` and by running `composer update`:
+
+```json
+"require": {
+    "drupal/entity_reference_revisions": "~1.3",
+    "drupal/composite_reference": "~1.0-alpha2",
+    "drupal/inline_entity_form": "~1.0-rc7",
+    "drupal/field_group": "~3.0",
+}
+```
+
+The `inline_entity_form` module requires the following patches to be applied:
+
+```json
+"patches": {
+    "drupal/inline_entity_form": {
+        "https://www.drupal.org/project/inline_entity_form/issues/3162384": "https://www.drupal.org/files/issues/2020-08-13/fixed_duplicate_rows-3162384-16.patch",
+        "https://www.drupal.org/project/inline_entity_form/issues/2842744#comment-13775778": "https://www.drupal.org/files/issues/2020-08-04/inline_entity_form-no_label_required_field_with_no_entries-2842744-27-D8.patch"
+    }
+}
+```
+
+The `field_group` module requires the following patches to be applied:
+
+```json
+"patches": {
+    "drupal/field_group": {
+        "https://www.drupal.org/project/field_group/issues/2787179#comment-13467953": "https://www.drupal.org/files/issues/2020-02-17/2787179-highlight-html5-validation-45.patch"
```

Comment:
This patch is no longer up to date, check the latest version in the composer json.

Reply:
I updated the patch reference.